drivers/stepper/step_dir/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Fabian Blatz <fabianblatz@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config STEP_DIR_STEPPER
+	bool
+	help
+	  Enable library used for step direction stepper drivers.
+
+config STEP_DIR_STEPPER_GENERATE_ISR_SAFE_EVENTS
```

Comment:
I think this feature could be used by other drivers as well, hence could be probably refactored out to Kconfig.stepper_event.template

Reply:
This could be then included over here with module called step_dir

Reply:
Yep, done.